graylog2-web-interface/src/stores/inputs/InputStatesStore.js

```diff
@@ -28,13 +28,53 @@ const InputStatesStore = Reflux.createStore({
             }
             result[input.id][node] = input;
           });
-        })
+        });
         this.inputStates = result;
         this.trigger({inputStates: this.inputStates});
 
         return result;
       });
-  }
+  },
+
+  start(input) {
+    let url;
+
+    if (input.global) {
+      url = URLUtils.qualifyUrl(jsRoutes.controllers.api.ClusterInputStatesController.start(input.id).url);
+    } else {
+      url = URLUtils.qualifyUrl(jsRoutes.controllers.api.InputStatesController.start(input.id).url);
+    }
+
+    return fetch('PUT', url)
+      .then(
+        () => {
+          this.list();
+          UserNotification.success(`Input '${input.title}' started successfully`);
```

Comment:
Nice catch. The problem is that the response from the server is not returning anything different when the input could start or not, so I think that's the first thing to change. Will improve that and update the PR with the changes :)